src/rgw/rgw_rest_s3.cc

```diff
@@ -3112,6 +3112,7 @@ void RGWPostObj_ObjStore_S3::send_response()
   dump_errno(s);
   if (op_ret >= 0) {
     dump_content_length(s, s->formatter->get_len());
+    dump_header_if_nonempty(s, "ETag", etag);
```

Comment:
Isn't etag already dumped a 3107?

Reply:
3107 means?

Reply:
https://github.com/ceph/ceph/blob/9809db9099d13986454e2f8279ab06a62a73131e/src/rgw/rgw_rest_s3.cc#L3107

Reply:
it dumped in the body. 

I found it also dumped in headers in AWS S3: https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPOST.html

```
HTTP/1.1 100 Continue
HTTP/1.1 200 OK
x-amz-id-2: LriYPLdmOdAiIfgSm/F1YsViT1LW94/xUQxMsF7xiEb1a0wiIOIxl+zbwZ163pt7
x-amz-request-id: 0A49CE4060975EAC
x-amz-version-id: default
Date: Wed, 12 Oct 2009 17:50:00 GMT
ETag: "1b2cf535f27731c974343645a3985328"
Content-Length: 0
Connection: close
Server: AmazonS3
```